tasks.py

```diff
@@ -33,7 +33,7 @@ def run_api(c):
 
 @task
 def run_worker(c):
-    c.run("python -m erica.infrastructure.rq.worker")
+    c.run("huey_consumer.py erica.application.JobService.job.huey -k thread -w 10")
```

Comment:
Probably a very basic question: where does `huey_consumer.py` come from? I didn't see it get added anywhere?

Reply:
It comes from the [huey package](https://github.com/coleifer/huey/blob/master/huey/bin/huey_consumer.py) that's why it was not added in this PR. I also did not find that very intuitive but it is [the documented way](https://huey.readthedocs.io/en/latest/consumer.html) to start the worker.